pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 88
-target-version = ['py35']
+target-version = ['py37']
```

Comment:
I think we should keep the target version at 3.6 (lowest we're going to support). We don't want to introduce 3.7 language features that some users may not have.